flow/project.py

```diff
@@ -3313,18 +3308,26 @@ def select(operation):
                 # Change groups to available run _JobOperation(s)
```

Comment:
What does this comment mean?

Reply:
Good question. I'm refactoring this code in a separate PR (not done yet). I will improve the code comments there.

Reply:
Fixed in #416.